node/technical-reference/overview.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Overview'
+description: 'Overview of the Atoma Node'
+---
+
+```mermaid
+graph TB
+    subgraph Atoma Node
+        AS[Atoma Service] --> |Handles API Requests| Auth[Authentication]
+        AS --> |Manages Models| CCS[Confidential Compute Service]
+        
+        SM[State Manager] --> |Manages State| DB[(PostgreSQL)]
+        
+        Sub[Sui Event Subscriber] --> |Events| SM
+        Sub --> |Blockchain Events| BC[Sui Blockchain]
+        
+        AD[Atoma Daemon] --> |Node Operations| SM
+        AD --> |Blockchain Operations| BC
+        
+        CCS --> |Secure Processing| Auth
+        Auth --> |Verify Access| SM
+    end
+
+    Client[External Client] --> |Purchase Stacks| BC[Sui Blockchain]
+    Client --> |API Requests| AS
+```
+
+## Component Overview
+
+### Core Components
+
+1. **Atoma Service**
+   - Main entry point for client requests
+   - Handles API endpoints for inference, embeddings, and image generation
+   - Manages request routing and response handling
+   - Integrates with external AI services
```

Comment:
What AI services are these ?

Mention also the authentication middleware layer, which is a crucial component of the service. The middleware allows your node to only process requests that were paid beforehand through the Atoma smart contract, as we well as being responsible for encrypt/decrypt requests (if confidential compute mode is enabled, running within a TEE).